docs/user_guide/7_restore_backup.md

```diff
@@ -0,0 +1,147 @@
+# Shadow Backup USB stick
+
+You can only follow these steps if you have been provided with a Shadow Backup USB stick. It has a 500 MB Clonezilla partition and a several GB for Clonezilla disk images. These can be used to restore any Shadow-provided laptop(s) and/or NUC(s) to their exact configuration (it's a full disk image restore) after the delivery was tested in Shadow's offices but before it was shipped to the customer. Doing this means losing all the data on the laptop(s) and/or NUC(s) since the original shipment of the equipment from Shadow.
+ 
+ The steps for restoring a NUC with Shadow Backup USB stick are the same as for laptop, except that the USB stick be connected to the NUC, as well as a monitor with HDMI connector (not provided) and a USB keyboard (not provided), and you have to choose a different image to restore from the relevant list in the Clonezilla process. Apart from that, the steps are the same.
+
+ You don't have to restore both NUC and laptop. You can choose to restore just 1 of them.
+
+## Clonezilla backup restore steps (here device means either laptop or NUC)
+
+1. Connect the Shadow USB stick to the device you want to restore
+
+2. Power on the device while tapping F2 and F12 on the keyboard. Disable Secure Boot in BIOS for the device by following the instructions [here](https://www.gigabyte.com/us/Support/FAQ/3001) (the menus might look like slightly different to these screenshots)
+
+3. Power on the device while tapping F7 and F8 on the keyboard. A blue dialog box will appear asking you which disk to boot from: select the Shadow backup USB 500 MB Clonezilla partition (it might show up as a SanDisk device, or a similar brand matching the model of the USB stick)
```

Comment:
Can you check if different F button needs to be pressed on laptop and nuc?

Reply:
Fixed in the last commit